camayoc/ui/models/components/add_new_dropdown.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from playwright.sync_api import TimeoutError
+
+from camayoc.exceptions import MisconfiguredWidgetException
+from camayoc.types.ui import UIPage
+
+
+class AddNewDropdown(UIPage):
+    def open_create_new_modal(self, type_ouiaid):
+        default_timeout = 5000  # 5s
```

Comment:
There's no very compelling reason for it.

Default Playwright timeout is 30 seconds and that's definitely too much. If dropdown did not appear shortly after clicking the button, it is not appearing.

There is a point by which dropdown should appear, but I do not know it. We could try to establish it with measurements, but in my experience OpenStack is pretty inconsistent. So it might be that dropdown appears within 10 ms, but sometimes OpenStack flounders and then dropdown appears after 100 ms. If we set up timeout to 50 ms, we risk a single test failing once in a blue moon and of course nobody is going to remember all these details and we won't be able to get down to it, so we just accept that _sometimes_ we need to rerun tests.

And to put things in perspective, I encountered that issue in maybe 3% of tests executed (I had few green runs before seeing it for the first time). So in large majority of cases Playwright will be clicking as fast as possible - we pay that 5 seconds waiting time only in these 3% of tests.

The specific 5 seconds I copied from [`items_list.py`](https://github.com/quipucords/camayoc/blob/main/camayoc/ui/models/components/items_list.py#L23). I might not have good reason to prefer 5 seconds over 1 second or 10 seconds, but at least I can make these magic numbers consistent.